cms-ajax.md

```diff
@@ -61,12 +61,14 @@ AJAX handlers are PHP functions that can be defined in the page or layout [PHP s
     url = "js"
     layout = "default"
     ==
+    <?php
```

Comment:
The <? ?> or <?php ?> tags are optional and are only necessary for code highlighting, they don't add value to describing an ajax handler